app/model.py

```diff
@@ -19,7 +19,12 @@ def sort_mentors(mentor: Dict) -> Tuple:
             )
 
         results = sorted(
-            self.db.search("Mentors", mentee["tech_stack"]),
+            self.db.read("Mentors", {
+                "tech_stack": mentee["tech_stack"],
+                "is_active": True,
+                "accepting_new_mentees": True,
+                "validate_status": 'approved'
+            }),
```

Comment:
Great, we've avoided the potential merge conflict from yesterday. Except for a humiliating indentation error that we should all be ashamed of, this was done perfectly. Happy to approve -- this is going to be a fantastic match mentees feature and will really pop whence visualized. Excited.